usaspending_api/local_settings.example.py

```diff
@@ -0,0 +1,9 @@
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'usaspending_api',
+        'USER': 'usaspending_api',
+        'PASSWORD': '',
+    }
+}
```

Comment:
Not a big issue, but something to consider earlier on -- I get the sense that the `local_settings` approach is now frowned on in favor of environment variables (in part due to the [12-factor push](https://12factor.net/)). In the database scenario, we tend to use [`dj-database-url`](https://github.com/kennethreitz/dj-database-url) on cloud.gov projects; it effectively expands a single database string into these four fields.